css/team.css

```diff
@@ -1,64 +1,155 @@
-/* background */
-.team{
+.team {
     background-color: #fff;
-    height: 100%;
     width: 100%;
 }
 
-.teamDescription{
+.team__title {
     color: #324B4C;
     font-family: 'Roboto', sans-serif;
-    font-size: 1rem;
+    font-size: 2rem;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+.team__person {
     text-align: center;
-    padding: 0 auto;
 }
 
-.teamName{
+.team__person--name {
     color: #324B4C;
     font-family: 'Roboto', sans-serif;
     font-size: 1.1rem;
     text-align: center;
     text-transform: capitalize;
 }
 
-.teamPhoto{
+.team__person--description {
+    color: #324B4C;
+    font-family: 'Roboto', sans-serif;
+    font-size: 1rem;
+    text-align: justify;
+    padding: 0 auto;
+}
+
+.team__clip-path-container {
+    display: inline-block;
+    position: relative;
+    height: 22vh;
+    width: 22vh;
+    margin: 10px ;
+    padding: 3px;
+    cursor: pointer;
+    transition: all .2s ease-in-out;
+    overflow: hidden;  
     border-radius: 50%;
-    width: 100%;
+    border-collapse: separate; 
+    background-color: #222;
 }
 
-.team__title{
-    color: #324B4C;
-    font-family: 'Roboto', sans-serif;
-    font-size: 2rem;
+.team__clip-path-container:focus,
+.team__clip-path-container:active,
+.team__clip-path-container:hover {
+    border-radius: 50%;
+    outline: none;
+} 
+
+.team__person--photo {
+    height: 22vh;
+    width: 22vh;
+    background-size: cover;
+    transition: all .2s ease-in-out;
+    transform: scale(1.4); 
+    outline: none;
+    overflow: hidden;  
+    clip-path: polygon(0 100%, 0 0, 52% 0, 52% 100%);
+    clip-path: polygon(0 100%, 0 0, 100% 0, 100% 100%);
+}
+
+.team__clip-path-container:hover .team__person--photo {
+    clip-path: polygon(50% 100%, 50% 0, 100% 0, 100% 100%);
+    transform: scale(1.1);
+
+}
+
+.team__clip-path-container:hover .team__person--links { 
+    left: 20px;
+    opacity: 1;
+}
+  
+.team__person--links {
+    position: absolute;
+    top: 4vh;
+    left: 120px;
+    height: 15vh;
+    width: 9vh;
+    transition: all .4s ease-in-out;
+    transition-delay: .1s;
+    opacity: 0;
     text-align: center;
-    text-transform: uppercase;
+}
+  
+.fab {
+    font-size: 6vh;
+    color: white;
+}
+
+.fab:hover {
```

Comment:
Nie działa we właściwy sposób. Nie podmienia koloru nie ten zielono/niebieski.

Reply:
Hmm... dziwne, nic w kodzie nie ruszalem i dziala u mnie:
![image](https://user-images.githubusercontent.com/29073258/52379334-8226ed80-2a6a-11e9-9c65-88052e917ffd.png)

Reply:
u mnie działa prawidłowo